code/modules/mob/living/carbon/life.dm

```diff
@@ -602,14 +602,15 @@ GLOBAL_LIST_INIT(ballmer_windows_me_msg, list("Yo man, what if, we like, uh, put
 
 /mob/living/carbon/proc/handle_bodycheck()
 	for(var/mob/H in oview(src, 7))
```

Comment:
```suggestion
	for(var/mob/living/L in oview(src, 7))
```
Or possible carbon or human, depending. Do you want people to feel sad about a dead monkey or corgi belonging to the same faction? If only about humans, then use 	`for(var/mob/living/carbon/human/H in oview(src, 7))` instead.